Documentation/criu.txt

```diff
@@ -213,6 +213,9 @@ In other words, do not use it unless really needed.
 *-s*, *--leave-stopped*::
     Leave tasks in stopped state after checkpoint, instead of killing.
 
+*--no-resume-on-error*::
+    Leave tasks in stopped state even if checkpoint completed unsuccessfully.
+
```

Comment:
Could you add some lines about what situations it is useful and why? What benefit does it provide?
How should I as a user decide what is preferred for them in case of criu failure?
Current doc lines are copy'n'paste from above without adding any useful context.